### PR TITLE
GAR GLASSES

### PR DESCRIPTION
**Pull request by dannno**
you're welcome

not all of them are in lockers/vendors because i didn't think it was appropriate
mappers can hide them around places if they want idk

### CODE REVIEW

**Comment by GunHog** [comment]
NT STATION RISES AGAIN

**Comment by TZK13** [comment]
Robot Roller-Derby Disco Dodgeball, a retail game on steam, includes gar glasses :^)

**Comment by TZK13** [comment]
Robot Roller-Derby Disco Dodgeball, a retail game on steam, includes gar glasses and they haven't gotten in trouble for it so :+1: 

:^)

**Comment by Razharas** [comment]
*ding ding ding ding
COPYRIGHT ALERT COPYRIGHT ALERT

**Comment by Nienhaus** [comment]
They have become a generic style of glasses now.

File: code/modules/clothing/glasses/glasses.dm [hunk 1]
```diff
@@ -78,6 +88,42 @@
 	flash_protect = 1
 	tint = 1
 
+/obj/item/clothing/glasses/sunglasses/garb
+	desc = "Go beyond impossible and kick reason to the curb!"
```
Comment (GunHog) [inline code/modules/clothing/glasses/glasses.dm]: byond*

**Comment by Cheridan** [comment]
I don't know what you're talking about, Razharas. These are merely cool bro shades that are worn by cool guys.

**Comment by Nienhaus** [comment]
@Cheridan ♥

**Comment by Razharas** [comment]
Ive seen it in anime so it must be copyright

**Comment by Nienhaus** [comment]
What's anime, Never heard of it.

**Comment by Cheridan** [comment]
Pretty sure Razharas is just making things up at this point.

**Comment by dannno** [comment]
spiral knights also features glasses of a similar styling with a much less subtle name and it's fine

**Comment by Supermichael777** [comment]
make them mesons available from mining vendor. Mine is the drill that will pierce the heavens!

**Comment by dannno** [comment]
you're a fucking genius kid
thank you

**Comment by francinum** [comment]
I cannot believe I didn't make that joke.

**Comment by dannno** [comment]
if someone tells me how embedding works i can add that as well

**Comment by Iamgoofball** [comment]
deep fryer when?

**Comment by WJohn** [comment]
One hell of a late port, but :+1:

**Comment by dannno** [comment]
yeah, I always assumed they would be :-1: because copyright
but they're allowed so eyyyyyyyyy

**Comment by dannno** [comment]
@RemieRichards what would I need to do to make these embed

**Comment by RemieRichards** [comment]
@dannno 
Find ```can_embed()```
Add them to that list (or add them to one of the ```is_sharp()``` lists.

Then if you want them to embed when THROWN, set their throwspeed to 4.

**Comment by Chocobro** [comment]
what the heck are gar glasses
:+1:

**Comment by dannno** [comment]
I guess @Nienhaus @WJohn ?

**Comment by dannno** [comment]
arigato

**Pull request merged**